package.json

```
@@ -3,7 +3,7 @@
   "private": true,
   "version": "0.0.1",
   "description": "Docs for Discordeno",
-  "repository": "https://github.com/discordeno/guide",
+  "repository": "https://github.com/discordeno/discordeno",
```

```suggestion
  "repository": "https://github.com/discordeno/guide"
```
This is used to generate the edit urls